californium-core/src/main/java/org/eclipse/californium/core/observe/InMemoryObservationStore.java

```diff
@@ -48,7 +48,7 @@
 	private volatile boolean enableStatus;
 
 	public InMemoryObservationStore(NetworkConfig config) {
-		int healthStatusInterval = config.getInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 60); // seconds
+		int healthStatusInterval = config.getInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 0); // seconds
```

Comment:
And yes, adding the explicit default should have been done already earlier by me. But maybe, you can add it now :-).